dashboard/app/models/programming_expression.rb

```diff
@@ -52,11 +55,24 @@ def self.properties_from_file(path, content)
         programming_environment_id: programming_environment.id,
         category: expression_config['category'],
         color: ProgrammingExpression.get_category_color(expression_config['category']),
-        parameters: expression_config['paletteParams']
+        syntax: syntax
       }
     end
   end
 
+  def self.get_syntax(config)
+    syntax = config['func']
+    if config['paletteParams']
+      syntax = config['func'] + "(" + config['paletteParams'].join(', ') + ")"
+    elsif config['block']
+      syntax = config['block']
+    elsif config['syntax']
+      syntax = config['syntax']
```

Comment:
what do you think about putting `syntax` first? it seems that if I needed `block` and/or `palletteParams` for something else, this suggestion would let me override their effect on how the block appears on our docs.

Reply:
Sure. Happy to change that to be first